woocommerce/woocommerce-ios
# Navigate to Refund Details screen

This PR updates the refund cell to the latest design and navigates to a blank refund details page after tapping on the cell.

| Refunded cells in Order Details | Refunded Details |
| --- | --- |
|<img src="https://user-images.githubusercontent.com/1062444/71592243-1e2f6c80-2af5-11ea-972f-58b5c2b316e4.png" width="350" /> | <img src="https://user-images.githubusercontent.com/1062444/71592260-30110f80-2af5-11ea-96ac-443dde8fa44d.png" width="350" />|

### To test
0. On the website, have an order with 1 or more refunds
1. In the app, navigate to Orders > Your chosen order > and scroll down to the "Payment" section in the order details.
2. Tap on a refund cell
3. It should navigate you to a blank Refund Details screen. The title should include the refund ID number and have a functioning back button (with no "back" title).

Update release notes:
